examples/rules/google-tags-fullstory.json

```diff
@@ -11,6 +11,21 @@
       "readOnLoad": true,
       "monitor": true
     },
+    {
+      "id": "fs-gtg-event",
+      "source": "dataLayer",
+      "operators": [
+        { "name": "query", "select": "$[(0,1,2)]" },
+        { "name": "query", "select": "$[?(0=event)]" },
+        { "name": "flatten" },
+        { "name": "query", "select": "$[?(0=event)]" },
```

Comment:
Is this `query` using an array index?!

Reply:
This where it's interesting, the `Arguments` object is essentially an array.  What we do is pick properties `0`, `1`, `2`, which has the effect of getting the values at array index 0-2.  That returns a POJO with 

```
{
 "0": <the 0 element>,
 "1": <the  1 element>,
 "2": <the 2 element>,
}
```

It then queries the "0" property, which is set to the string `event`.